### PR TITLE
fixed some typo in docs

### LINKED ISSUE #5472: Original price calculation typo
## Current behavior

Original Product Price calculation typo.
https://github.com/vuestorefront/vue-storefront/blob/master/core/modules/catalog/helpers/price/syncProductPrice.ts line 15
`  product.original_price_tax = product.original_price_incl_tax - product.original_trice`
original_**trice**

## Expected behavior

`  product.original_price_tax = product.original_price_incl_tax - product.original_price`

## Version of Vue Storefront

- [x] Vue Storefront
- [ ] Vue Storefront Next

## Can you handle fixing this bug by yourself?

- [ ] YES
- [ ] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer. (doesn't apply to Next)
Pick one option.

- [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [ ] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [ ] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details

- Browser: 
- OS: 
- Node: 
- Code Version:

### PR DESCRIPTION
**Pull request by lukaszjedrasik**
### Related Issues

closes #5472 

### Short Description and Why It's Useful
As we talked, I have used changes from this PR https://github.com/vuestorefront/vue-storefront/pull/5391

### Which Environment This Relates To
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog
- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature
- [ ] (Next only) I've followed this [instruction](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and I've created a `.js` file with information about my Pull Request

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by Fifciu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Got rid of inifnity redirect or page-not-found on refresh category/product view  
 - Got rid of memory leak related to dynamic config - tomasz-duda (#4768)
 - servercart-after-diff event payload - Fifciu (#5365)
+- Fixed some typo in docs (#5071)
```
[the remark this answers is not in the document]
Comment (Fifciu) [reply CHANGELOG.md:40]: Please also mark yourself and @akucharczyk

**Review by Fifciu — CHANGES_REQUESTED** [review]
Changes requested

**Comment by lukaszjedrasik** [comment]
@Fifciu fixed

**Review by Fifciu — APPROVED** [review]
Look great, good job

**Pull request merged**